### PR TITLE
fix(upload): fallback sha1 calculation for IE11 using new package

### PR DESCRIPTION
**Pull request by omgrief**

### CODE REVIEW

File: src/utils/__tests__/webcrypto.test.js [hunk 1]
```diff
@@ -64,13 +66,33 @@ describe('util/webcrypto', () => {
                     },
                 };
 
-                const expectedError = new Error('lol');
+                const expectedError = new Error('ERROR');
```
Comment (sahiga) [inline src/utils/__tests__/webcrypto.test.js]: Thank you for fixing this

File: src/utils/__tests__/webcrypto.test.js [hunk 2]
```diff
@@ -64,13 +66,33 @@ describe('util/webcrypto', () => {
...
+                window.msCrypto = {
+                    subtle: {
+                        digest: digestMock,
+                    },
+                };
+
+                const hash = await digest('SHA-1', buffer);
+                expect(hash).toBeDefined();
+                expect(digestMock).not.toHaveBeenCalled();
+                expect(sha1.arrayBuffer).toHaveBeenCalledWith(buffer);
```
Comment (sahiga) [inline src/utils/__tests__/webcrypto.test.js]: Can we also test (1) that `sha1.arrayBuffer()` is not called when the algorithm is not SHA-1 and (2) that the promise is rejected when the call to `sha1.arrayBuffer1()` fails?

File: src/utils/__tests__/webcrypto.test.js (hunk 1 above)
Comment (omgrief) [reply src/utils/__tests__/webcrypto.test.js]: 👍

File: src/utils/__tests__/webcrypto.test.js (hunk 2 above)
Comment (omgrief) [reply src/utils/__tests__/webcrypto.test.js]: added tests

**Comment by alexkrolick** [comment]
This commit is missing the BREAKING CHANGE keyword in the message, we''ll have to do something to trigger that in the autogenerated release

**Pull request merged**